src/utils.ts

```diff
@@ -57,7 +59,14 @@ export function getTag(name: string, isVideo: boolean): MediaType {
 					? MediaType.ANIME
 					: MediaType.OTHER;
 }
-
+export function determineSkipRecheck(decision: Decision): boolean {
+	const { skipRecheck } = getRuntimeConfig();
+	return decision === Decision.MATCH_PARTIAL
+		? false
+		: decision === Decision.MATCH
+			? true
+			: skipRecheck;
+}
```

Comment:
should we just get rid of skipRecheck being configurable at all?

Reply:
In favor of always rechecking risky?

I use `true,` I've had maybe a handful of false positives due to trackers uploading corrupted stuff out of many thousands of matches.

I don't imagine people's experiences would necessarily align with mine given the different level of consistencies amongst tracker's content though.

Reply:
always rechecking risky, or never rechecking risky. I don't really care. I just think that the program understands whether to recheck better than the user does.

Reply:
If we are to remove the option this is what I think:
* MATCH: Skip rechecking and start unpaused
* MATCH_SIZE_ONLY: Skip rechecking and start unpaused (should error if something is wrong? If not recheck and pause)
* MATCH_PARTIAL: Recheck and pause

Reply:
>(should error if something is wrong? If not recheck and pause)

What do you mean if something is wrong, what would we measure this by?

Reply:
If it throws MISSING_FILES or some other error. But are doing recheck and pause of this.

Reply:
I'm of the mind that, and haven't ever used `skipRecheck: false`, that rechecking even risky at this point is generally unnecessary; that we should only be rechecking partials.

Given the filtering we are doing in decide it should be _VERY_ edge-case that we get a match injected that is not correct. The one caveat being if a tracker uploads corrupted files.

Reply:
The failure mode I'm worried about here is rechecking and starting a torrent that turns out to not actually be a match and incurring a large DL on that torrent. As long as whenever we recheck we also pause (which I think we're doing), the impact of this decision is minimized and leaves the door open to changes later. I'm fine either way here.

Reply:
Yes, anytime rechecks are done in deluge or qbittorrent it is left paused, which prevents any incurred download.

The one edge-case is if pieces do not match in qbittorrent, the behavior I remember in that case is that individually, when pieces are requested and determined to be different, it will request the pieces and subsequently download them and replace the originals.

This is generally only going to be a problem in the case I mentioned, corrupted uploads of a torrent that is otherwise the same size and passes through decide.